ccd-definition/CaseType.json

```diff
@@ -2,8 +2,8 @@
   {
     "LiveFrom": "01/01/2017",
     "ID": "Shared_Storage_DRAFTType",
```

Comment:
Please change case ID

Reply:
If not deployment to production will override current shared storage POC.

Reply:
agreed planned and raised raised as separate task